src/Controls/src/Core/Slider/Slider.iOS.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Maui.Controls.Platform;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+
+namespace Microsoft.Maui.Controls
+{
+	public partial class Slider
+	{
+		public static void MapUpdateOnTap(SliderHandler handler, Slider slider) =>
```

Comment:
Can we just make all public APIs added in this PR internal? 

So, we're not adding any new API?